swc/defs.bzl

```diff
@@ -100,17 +100,22 @@ def swc(name, srcs, args = [], data = [], plugins = [], output_dir = False, swcr
     # Determine js & map outputs
     js_outs = []
     map_outs = []
+    dts_outs = []
 
     if not output_dir:
-        js_outs = _swc_lib.calculate_js_outs(srcs, out_dir, root_dir)
-        map_outs = _swc_lib.calculate_map_outs(srcs, source_maps, out_dir, root_dir)
+        if kwargs.get("experimental_emit_isolated_dts", False):
+            dts_outs = _swc_lib.calculate_dts_outs(srcs, out_dir, root_dir)
```

Comment:
can it produce d.ts.map files too, like `tsc` does?

Reply:
From what I can find it seems like swc can do one or the other right now, not both. If you pass `--out-file` more then once it's an error, if you don't pass it at all it outputs to stdout instead...

Reply:
Wait, I read that wrong... I meant it can't produce .js and .d.ts, I'll look into .d.ts.map more